content/documentation/pages/3-stream-developer-guides/2-streams/1-standalone-stream-sample.md

```diff
@@ -0,0 +1,641 @@
+---
+path: 'stream-developer-guides/streams/standalone-stream-sample/'
+title: 'Stream Application Development'
+description: 'Create your own microservices for Stream processing and deploy them manually'
+---
+
+In this guide, we develop three Spring Boot applications that use Spring Cloud Stream and deploy them to Cloud Foundry, to Kubernetes, and to your local machine.
+In another guide, we [deploy these applications by using Data Flow](%currentPath%/stream-developer-guides/streams/data-flow-stream/).
+By deploying the applications manually, you get a better understanding of the steps that Data Flow automates for you.
+
+The sections that follow describe how to build these applications from scratch.
+
+If you prefer, you can download a zip file containing the sources for these applications, unzip it, build it, and proceed to the [deployment](#deployment) section.
+
+You can [download the project](https://github.com/spring-cloud/spring-cloud-dataflow-samples/blob/master/dataflow-website/stream-developer-guides/streams/standalone-stream-sample/dist/usage-cost-stream-sample.zip?raw=true) that contains all three applications from your browser. You can also use the command line, as the following example shows:
```

Comment:
Link for `download the project` returns a 404.

Reply:
I believe it will only be available after we push the changes.

Reply:
Right, the projects merged for DRY